pom.xml

```diff
@@ -3,7 +3,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.609.3</version>
```

Comment:
Sure. In this issue [JENKINS-28550](https://issues.jenkins-ci.org/browse/JENKINS-28550), and its [PR](https://github.com/jenkinsci/jenkins/pull/1717). This PR was release in 1.619.